consensus/poet/core/sawtooth_poet/poet_consensus/poet_config_view.py

```diff
@@ -51,6 +51,16 @@ def __init__(self, state_view):
 
         self._config_view = ConfigView(state_view)
 
+        self._block_claim_delay = None
```

Comment:
As long as the lifetimes of instances of this class are short, this is probably fine.  If this is too-long lived, changes in these values that occur between block commits will get lost.